server/services/employers.js

```diff
@@ -11,6 +11,23 @@ const getEmployers = async (user) => {
 
 const getEmployerByID = async (id) => dbClient.db[collections.EMPLOYER_FORMS].findDoc({ id });
 
+const saveSingleSite = async (siteJson) => {
+  const res = await dbClient.db.saveDoc(collections.EMPLOYER_SITES, siteJson);
+  const { siteId } = res;
+  switch (res.status) {
```

Comment:
this is where things are falling over, but I can't debug locally - might need to change the way I'm attempting to save the record on line 15. I do want to be able to distinguish duplicates from other failure reasons and then have server.js return error codes the FE can use. I'm think we'll want to be able to let the user know while they're still in the modal (so if it's a dupe, we can keep the modal open with a failure for the dupe in case they just made a typo)